crates/re_viewer/src/app.rs

```diff
@@ -744,6 +746,12 @@ impl App {
         egui_ctx.set_style((*style).clone());
     }
 
+    fn log_db_is_nonempty(&self) -> bool {
+        self.log_dbs
+            .get(&self.state.selected_rec_id)
+            .map_or(false, |log_db| !log_db.is_empty())
+    }
```

Comment:
`is_nonempty` feels very non-idiomatic and double-negativist... `false` means it's not non-empty?

Reply:
I agree, but I failed to find something better. Negating it we get something like `log_db_is_empty_or_there_is_no_log_db`

Reply:
I added an explanatory docstring that might help.